Assets/Scripts/Localization/LocalizationTable.cs

```diff
@@ -120,6 +174,82 @@ public static string[] GetLanguages()
             return localizationTable.Keys.ToArray();
         }
 
+        public static void LoadConfigFile(string pathToConfigFile)
+        {
+            localizationConfigurations = new Dictionary<string, LanguageData>();
+
+            if (File.Exists(pathToConfigFile) == false)
+            {
+                Debug.ULogErrorChannel("LocalizationTable", "No config file found at: " + pathToConfigFile);
```

Comment:
Maybe a note of something to the effect about needing to stop and restart in the error message? I don't know exactly, I'm just expecting ending up with lots of bug reports during development any time someone goes to do something while not having config.xml that would be fixed by just stopping and hitting play again after it's downloaded a new one.